lib/perl/Genome/Model/Tools/DetectVariants2/Dispatcher.pm

```diff
@@ -261,18 +261,18 @@ sub _dump_workflow {
     my $self = shift;
     my $workflow = shift;
     my $xml = $workflow->get_xml();
-    my $xml_location = $self->output_directory."/workflow.xml";
+    my $xml_location = $self->_temp_staging_directory."/workflow.xml";
     $self->_rotate_old_files($xml_location); #clean up any previous runs
     my $xml_file = Genome::Sys->open_file_for_writing($xml_location);
     print $xml_file $xml;
     $xml_file->close;
-    #$workflow->as_png($self->output_directory."/workflow.png"); #currently commented out because blades do not all have the "dot" library to use graphviz
+    #$workflow->as_png($self->_temp_staging_directory."/workflow.png"); #currently commented out because blades do not all have the "dot" library to use graphviz
 }
 
 sub _dump_dv_cmd {
     my $self = shift;
     my $cmd = join(" ",@INC)."\n===============================================\n";
-    $cmd .=   "gmt detect-variants2 dispatcher --output-directory ".$self->output_directory;
+    $cmd .=   "gmt detect-variants2 dispatcher --output-directory ".$self->_temp_staging_directory;
```

Comment:
I don't believe this one should have been updated.  This "dump" is meant to represent what the user would type in to run this dispatcher again.